src/options/pages/blueprints/BlueprintsCard.tsx

```diff
@@ -157,7 +157,7 @@ const BlueprintsCard: React.FunctionComponent<{
   return (
     <BootstrapRow className={styles.root}>
       <ListFilters teamFilters={teamFilters} tableInstance={tableInstance} />
-      <Col className={styles.mainContainer}>
+      <Col className={styles.mainContainer} md={9} sm={12} xl={10}>
```

Comment:
NIT: reorder the attributes so they got from the smallest to the largest (or vice versa):
```suggestion
      <Col className={styles.mainContainer} sm={12} md={9} xl={10}>
```